lib/dom.dart

```diff
@@ -2,10 +2,13 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-// TODO(joshualitt): Import js types properly.
-import 'dart:typed_data';
+import 'dart:js_interop';
```

Comment:
I'm genuinely surprised this lets you do this. It seems like Dart allows you to shadow exports? They have two `@JS` annotations and they're separate semantics (the `dart:js_interop` does type checks), so I think you want to `hide` the `package:js` one here and everywhere else.

I need to think about this a bit more because I was hoping there would be a static error when you import both, not whatever this is.